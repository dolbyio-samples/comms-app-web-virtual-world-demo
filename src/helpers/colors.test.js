import { describe, it, expect } from "vitest";
import { convertStringToHexaColor } from "./colors";

describe("colors test suite", () => {
  it("color should match the snapshot", () => {
    const color = convertStringToHexaColor("test color");
    expect(color).toMatchSnapshot();
  });
});
