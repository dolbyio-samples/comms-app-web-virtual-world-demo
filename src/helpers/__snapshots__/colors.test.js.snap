// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`colors test suite > color should match the snapshot 1`] = `"#555203"`;
